README.md

```diff
@@ -18,26 +17,66 @@ in Python without C++.
 
 Halide requires C++17 (or later) to use.
 
-For more detail about what Halide is, see http://halide-lang.org.
+For more detail about what Halide is, see https://halide-lang.org.
 
-For API documentation see http://halide-lang.org/docs
+For API documentation see https://halide-lang.org/docs.
 
-To see some example code, look in the tutorials directory.
+For some example code, read through the tutorials online
+at https://halide-lang.org/tutorials. The corresponding code is in the
+`tutorials/` directory. Larger examples are in the `apps/` directory.
 
 If you've acquired a full source distribution and want to build Halide, see the
-[notes below](#building-halide-with-cmake).
+[notes below](#building-halide).
 
 # Getting Halide
 
+## Pip
+
+As of Halide 19.0.0, we provide binary wheels on PyPI which include the Python
```

Comment:
It's weird to use pip to get a c++ dependency, so add something like: "Halide provides bindings for C++ and python. Even if you only intend to use Halide from C++, pip may be the easiest way to get a binary build of Halide."

Reply:
Good suggestion! I reworked the language here to include what you wrote.